src/test/resources/database/bag-index.sql

```diff
@@ -1,8 +1,9 @@
 create table bag_info (
     bagId text not null unique,
     base text not null,
-    created text not null);
+    created text not null,
+    doi text not null);
```

Comment:
the dog is not unique in this schema and you even add the same one twice, is there a valid case for this which has to do with versions or am I completely lost here?